src/app/projects/components/project-settings/project-settings.component.ts

```diff
@@ -555,18 +555,12 @@ export class ProjectSettingsComponent implements OnInit, OnDestroy, AfterViewIni
 
   focusModalInputBox(event: Event, inputBoxName: string) {
     event.preventDefault();
-    const inputChildren = document.getElementsByName(inputBoxName);
-    if (inputChildren.length > 0) {
-      const node = inputChildren[0];
-      if (
-        node instanceof HTMLElement &&
-        node.getAttribute('name') == inputBoxName
-      ) {
-        setTimeout(() => {
-          node.focus();
-        }, 0);
-        return;
-      }
+    const inputChildren = document.getElementById(inputBoxName);
+    if (inputChildren && inputChildren instanceof HTMLElement && inputChildren.getAttribute('name') == inputBoxName) {
+      setTimeout(() => {
+        inputChildren.focus();
+      }, 0);
+      return;
```

Comment:
since we have an id we don't need to check the name so id would go with something like this:

```Typescript

  focusModalInputBox(event: Event, inputBoxId: string) {
    event.preventDefault();
    const inputElement = document.getElementById(inputBoxId) as HTMLInputElement;
    if (inputElement) inputElement.focus();
  }
```

no clue why there is a timeout there, I think we don't need it anymore.